src/main/java/co/omise/models/Account.java

```diff
@@ -5,50 +5,72 @@
 import co.omise.requests.ResponseType;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import okhttp3.HttpUrl;
+import okhttp3.RequestBody;
+
+import java.io.IOException;
+import java.util.List;
 
 /**
- * Represents Omise Account object and contains its {@link RequestBuilder}.
+ * Account object
  *
  * @see <a href="https://www.omise.co/account-api">Account API</a>
  */
 public class Account extends Model {
-    private String email;
     private String currency;
+    private String email;
+    private String location;
+    @JsonProperty("supported_currencies")
+    private List<String> supportedCurrencies;
     @JsonProperty("team")
     private String teamId;
 
-    public Account() {
+    public String getCurrency() {
+        return this.currency;
+    }
+
+    public void setCurrency(String currency) {
+        this.currency = currency;
     }
 
     public String getEmail() {
-        return email;
+        return this.email;
```

Comment:
I see that `this.` is added to every getter. Is this done with a purpose or is it just some coding style?